gobind is not in the path

I have to add $GOPATH/bin into path manully. I thought this should be added into the makefile since the gomobile is installed with make.

I'm using go 1.18.3. I got

```
touch go.mod
env GOBIN=/builds/linsui/fdroiddata/build/srclib/outline-go-tun2socks/bin go install golang.org/x/mobile/cmd/gomobile
/builds/linsui/fdroiddata/build/srclib/outline-go-tun2socks/bin/gomobile init
mkdir -p /builds/linsui/fdroiddata/build/srclib/outline-go-tun2socks/build/intra
/builds/linsui/fdroiddata/build/srclib/outline-go-tun2socks/bin/gomobile bind -a -ldflags '-w' -target=android -tags android -work -o /builds/linsui/fdroiddata/build/srclib/outline-go-tun2socks/build/intra/tun2socks.aar github.com/Jigsaw-Code/outline-go-tun2socks/intra github.com/Jigsaw-Code/outline-go-tun2socks/intra/android github.com/Jigsaw-Code/outline-go-tun2socks/intra/doh github.com/Jigsaw-Code/outline-go-tun2socks/intra/split github.com/Jigsaw-Code/outline-go-tun2socks/intra/protect
WORK=/tmp/gomobile-work-3757592961
/builds/linsui/fdroiddata/build/srclib/outline-go-tun2socks/bin/gomobile: gobind was not found. Please run gomobile init before trying again
Makefile:17: recipe for target '/builds/linsui/fdroiddata/build/srclib/outline-go-tun2socks/build/intra/tun2socks.aar' failed
make: *** [/builds/linsui/fdroiddata/build/srclib/outline-go-tun2socks/build/intra/tun2socks.aar] Error 1
```

when I ran `make intra`. I install go with

```
curl -Lso go.tar.gz https://go.dev/dl/go1.18.3.linux-amd64.tar.gz
echo "956f8507b302ab0bb747613695cdae10af99bbd39a90cae522b7c0302cc27245 go.tar.gz" | sha256sum -c -
tar xzf go.tar.gz --strip-components=1 -C /usr/local/
```

on Debian Stretch.

`PATH="$HOME/go/bin:$PATH" make intra` works.

--- a/Makefile
+++ b/Makefile
@@ -9,7 +9,8 @@ IMPORT_PATH=github.com/Jigsaw-Code/outline-go-tun2socks
 all: intra android linux apple windows
 
 # Don't strip Android debug symbols so we can upload them to crash reporting tools.
-ANDROID_BUILD_CMD=$(GOMOBILE) bind -a -ldflags '-w' -target=android -tags android -work
+# Add GOBIN to $PATH so `gomobile` can find `gobind`.
+ANDROID_BUILD_CMD=env PATH=$(GOBIN):$(PATH) $(GOMOBILE) bind -a -ldflags '-w' -target=android -tags android -work
 
 intra: $(BUILDDIR)/intra/tun2socks.aar
 
@@ -59,7 +60,7 @@ $(WINDOWS_BUILDDIR)/tun2socks.exe: $(XGO)
 
 $(GOMOBILE): go.mod
 	env GOBIN=$(GOBIN) go install golang.org/x/mobile/cmd/gomobile
-	$(GOMOBILE) init
+	env GOBIN=$(GOBIN) $(GOMOBILE) init
 
 $(XGO): go.mod
 	env GOBIN=$(GOBIN) go install github.com/crazy-max/xgo

